fix(angular): mention `TraceDirective` bug and workaround in Angular SDK (#4780)
getsentry/sentry-docs: src/platforms/javascript/guides/angular/components/tracehelpers.mdx

    }
  })
```

> 3. `TraceDirective` to track a duration between `OnInit` and `AfterViewInit` lifecycle hooks in template:

Reviewer: ```suggestion
3. `TraceDirective` tracks a duration between `OnInit` and `AfterViewInit` lifecycle hooks in template:
```

<Alert level="warning" title="Important">

> The usage of `TraceDirective` or `TraceModule` currently causes a compiler error at application compile

Reviewer: simplifying and taking it out of passive voice 

```suggestion
Using `TraceDirective` or `TraceModule` currently causes a compiler error at application compile
```

time of your Angular application if AOT compilation is enabled in your application config (which it is by default).
This is a [known limitation](https://github.com/getsentry/sentry-javascript/issues/3282) of our current
Angular SDK (v6.*) and it will be [adressed and fixed](https://github.com/getsentry/sentry-javascript/issues/4644)
in our next major Angular SDK release (v7).

> In the meantime, please refer to Options 1 (`TraceClassDecorator`) and 2 (`TraceMethodDecorator`)

Reviewer: again, just simplifying language where possible

```suggestion
In the meantime, please use options 1 (`TraceClassDecorator`) and 2 (`TraceMethodDecorator`)
```
